platform/unix/configure.ac

```diff
@@ -29,6 +29,10 @@ AC_DEFUN([LOVE_MSG_ERROR],
 # C++11 support in cpp11.m4
 ACLOVE_CPP11_TEST
 
+# Add -fvisibility=hidden and -fvisibility-inlines-hidden
+CFLAGS="-fvisibility=hidden $CFLAGS"
+CPPFLAGS="-fvisibility=hidden -fvisibility-inlines-hidden $CPPFLAGS"
```

Comment:
does `fvisibility-inlines-hidden` do anything when `-fvisibility=hidden` is already set?

Reply:
Yes. From https://gcc.gnu.org/wiki/Visibility

> This causes all inlined class member functions to have hidden visibility, causing significant export symbol table size & binary size reductions though not as much as using `-fvisibility=hidden`. However, `-fvisibility-inlines-hidden` can be used with no source alterations, unless you need to override it for inlines where address identity is important either for the function itself or any function local static data.

Reply:
Although from what I read it seems to _do nothing_ if `-fvisibility=hidden` is already set, my testing says otherwise as it reduces the size further _slightly_.

Reply:
Thanks!